### ISSUE TITLE
Design and develop a new tool to evaluate the quality of a set of search results

### ISSUE THREAD

**Issue by jjacob7734** [NASA-PDS/planetary-data-engine#15]
## 💡 Description

Design and develop a software tool that can assign a score to a set of search results to quantify the quality of the result set.  The inputs to the tool are a set of search results and an a labeling of the top 10 documents in the search result set on a scale of `0` to `5` where `0` means `completely irrelevant` and 5 means `extremely highly relevant`.  The expectation is that the documents of high relevance will appear at or near the top of the result set for a successful search.  Only the top 10 documents need to be considered for evaluation in this iteration.

**Comment by tloubrieu-jpl** [comment]
@jjacob7734 is making progress on this ticket with a single test test suite (Cassini search).

**Comment by jjacob7734** [comment]
The best metric I've found to evaluate the quality of a search technology is the Normalized Discounted Cumulative Gain (NDCG).  That metric and other common ones are succinctly described here:  https://ml-compiled.readthedocs.io/en/latest/metrics.html.

**Comment by jjacob7734** [comment]
I am using this implementation in the `scikit-learn` module: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.ndcg_score.html

**Comment by jordanpadams** [comment]
Not completed prior to project pause. Moving to icebox
